logic/规则Logic.md

```diff
@@ -438,7 +180,7 @@
 - 被动技能
   - 冥想
     -当玩家处于可接受指令状态且不在修机时，会积累学习进度，速度为（GameData.basicFixSpeed / 4）/ms
-    -受到攻击或眩晕或翻窗（或攻击他人）学习进度清零
+    -受到攻击（并非伤害）或眩晕或翻窗（或攻击他人）学习进度清零
 -主动技能
   - 写答案
     -使用瞬间，对于可互动范围内的一台电机增加这个学习进度
```

Comment:
I believe writing down your answers requires time, thus instead of  an 'instant' fix, I suggest using a speed of GameData.basicFixSpeed * 127 or sth of the sort